crates/typst/src/export/svg.rs

```diff
@@ -100,25 +173,36 @@ impl SVGRenderer {
     }
 
     /// Render a frame with the given transform.
-    fn render_frame(&mut self, frame: &Frame, ts: Transform) {
+    fn render_frame(&mut self, state: State, transform: Transform, frame: &Frame) {
         self.xml.start_element("g");
-        if !ts.is_identity() {
-            self.xml.write_attribute("transform", &SvgMatrix(ts));
+        if !transform.is_identity() {
+            self.xml.write_attribute("transform", &SvgMatrix(transform));
         }
 
         for (pos, item) in frame.items() {
+            // File size optimization
```

Comment:
This is good for now, but ideally we wouldn't wrap every single frame item in a `<g>`.

Reply:
I agree, it makes debugging SVGs a lot harder since there are so many nested level, but I don't want to accidentally break it all of now so I just added small file size optimizations where I stumbled on them.